src/Loading/errors.ts

```diff
@@ -0,0 +1,18 @@
+import { BaseError } from "../Misc/baseError";
+
+/**
+ * Application error to indicate load scene failures.
+ */
+export class SceneLoaderError extends BaseError {
+    /**
+     * Creates a new SceneLoaderError
+     * @param message defines the message of the error
+     * @param innerException the exception that caused the outer exception
+     */
+    constructor(message: string, innerException?: Error) {
+        super(message, undefined, innerException);
```

Comment:
Why no error code for this category of errors?

Reply:
This is supposed to be an outer error for what I have currently added. But on second thought, it would be good to have code for this.